docs/forensics/INDEX.md

```
@@ -7,6 +7,11 @@ This index enumerates evidence packs stored under `docs/forensics/`.
 | --- | --- | --- | --- | --- |
 | Hygiene | docs/forensics/evidence_hygiene_remediation_evidence.md | Evidence hygiene remediation proof pack | PR #15 / fa5d30c | pending |
 
+## Phase remediation evidence
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)